### PR TITLE
feat: Add versioning columns to alias content admin manager

### PR DESCRIPTION
**Pull request by NarenderRajuB**
1. Add versioning columns to Alias content Manager
2. Add Unpublished, Site and Category filter to the Alias content

### CODE REVIEW

**Review by adam-murray — CHANGES_REQUESTED** [review]
Could you add a CHANGELOG entry as well?

**Review by Aiky30 — CHANGES_REQUESTED** [review]

File: djangocms_alias/admin.py [hunk 1]
```diff
@@ -85,9 +95,18 @@ def delete_model(self, request, obj):
...
-    list_filter = (LanguageFilter,)
+    list_filter = (LanguageFilter, )
+    list_display = alias_content_admin_list_display
+    change_form_template = "admin/djangocms_alias/aliascontent/change_form.html"
+
+    # Add Alias category in the admin manager list and order field
+    def get_category(self, obj):
+        return obj.alias.category
+
+    get_category.short_description = 'category'
```
Comment (Aiky30) [inline djangocms_alias/admin.py:108]: This should be translatable as it is shown to the user

```suggestion
    get_category.short_description = _("category")
```

Comment (Aiky30) [reply djangocms_alias/admin.py:108]: Will need this adding to the top, the editor won't let me add it as a suggestion: 

`from django.utils.translation import gettext_lazy as _`

File: tests/test_admin.py [hunk 2]
```diff
@@ -51,3 +55,120 @@ def test_language_filter(self):
...
+            category=category,
+            position=0,
+        )
+        expected_en_content = AliasContent.objects.create(
+            alias=alias,
+            name="EN Alias Content",
+            language="en",
+        )
+        # If versioning is enabled be sure to create a version
+        if is_versioning_enabled():
```
Comment (Aiky30) [inline tests/test_admin.py:114]: We know that versioning is enabled for this test now. 

```suggestion
        if is_versioning_enabled():
```

File: tests/test_admin.py [hunk 3]
```diff
@@ -51,3 +55,120 @@ def test_language_filter(self):
...
+            response_content_decoded,
+        )
+        self.assertIn(
+            latest_alias_content_version.get_state_display(),
+            response_content_decoded,
+        )
+        self.assertIn(
+            localtime(
+                latest_alias_content_version.modified
+            ).strftime("%B %-d, %Y, %-I:%M %p").replace('AM', 'a.m.').replace('PM', 'p.m.'),
```
Comment (Aiky30) [inline tests/test_admin.py:172]: Manipulating a date string like this should not be required i.e. with the replace. The admin will get it's string format from the settings, you should be able to import this from the settings loaded to use that: 

https://docs.djangoproject.com/en/3.2/ref/settings/#datetime-format

**Review by Aiky30 — CHANGES_REQUESTED** [review]
Minor changes required. Alias is a very high level package so needs to be treated like the core of the CMS, the same for Versioning and URL Manager.

File: tests/test_admin.py (hunk 3 above)
Comment (NarenderRajuB) [reply tests/test_admin.py]: Code updated based on comments

**Review by Aiky30 — APPROVED** [review]

**Review by adam-murray — APPROVED** [review]

**Pull request merged**